mdx-dev/platform-ui-pdf-generator
# Adds legend to pharmacy pages

To test:

- do all requisite vagrant stuff to get you to the right repository
- gulp build
- head over to [the pdf page](http://vitalschoice.dev/pdf/#/)
- Scroll down to a page with pharmacies (page 108 by my count)
- Be filled with awe as you see that each pharmacy now has a teeny tiny letter underneath it indicating what kind of pharmacy it is.
- Marvel and be amazed at the legend that now appears at the bottom of the page. Should look like this:

<img width="1004" alt="screen shot 2015-11-19 at 3 08 55 pm" src="https://cloud.githubusercontent.com/assets/6492662/11282847/85399b88-8ecf-11e5-8b78-ec06b7890994.png">
